app/models/pipeline/logic_review.py

```diff
@@ -92,20 +92,11 @@ async def review(self, mr: MergeRequest) -> PipelineResult:
                 summary="解析审查响应失败", comments=[]
             )
 
-        # 创建评论，每个文件最多保留2条最重要的评论
         comments = []
-        file_comment_count = {}
         for ai_comment in ai_review.comments:
             if ai_comment.type == "praise":
```

Comment:
[From AI Code Reviewer][bot] Consider adding back the logic to limit the number of comments per file for conciseness.  While showing all comments might be useful in some cases, too many comments can overwhelm the user and make it difficult to focus on the most critical issues.